contrib/action_nitro/src/action_nitro.nit

```diff
@@ -561,7 +561,7 @@ redef class Player
 		end
 
 		# Display respawn instructions
-		app.ui_sprites.add new Sprite(app.texts_sheet.respawn, app.ui_camera.center)
+		app.ui_sprites.add new Sprite(app.texts_sheet.respawn, app.ui_camera.center.offset(0.0, 0.0, 0.0))
```

Comment:
The anchors are typed as `IPoint3d` now, they can't be used directly in a sprite anymore. This is the drawback of this PR, however, it is not usually an issue because we rarely lock a sprite to an anchor. Except for `center` which is used to display logos and such. Here I use `offset` as a kind of copy service.

An alternative would be to type `Sprite::center` as an `IPoint3d`, however, as of now, it doesn't have writing services (`x=`...) so it breaks a lot of clients. If this becomes a common issue, we could revisit this solution.